drivers/frequency/ad9508/ad9508.c

```diff
@@ -171,6 +171,8 @@ int32_t ad9508_setup(struct ad9508_dev **device,
 	if (ret != 0)
 		return ret;
 
+	*device = dev;
```

Comment:
the commit is good but the title/description is misleading, you're not creating the ad9508 device descriptor, that's already done, you make sure it's stored in the output variable

Reply:
Changed